### PR TITLE
Adding the link to the E2C migration guide

### PR DESCRIPTION
**Pull request by ishanjainn**
**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/introduction/grafana-enterprise.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ To learn more about Grafana Enterprise, refer to [our product page](/enterprise)
 
 Many Grafana Enterprise features are also available in [Grafana Cloud](/docs/grafana-cloud) Free, Pro, and Advanced accounts. For details, refer to [the Grafana Cloud features table](/pricing/#featuresTable).
 
+To Migrate from Grafana Enterprise to Grafana Cloud, Refer to our [Migration guide](/docs/grafana-cloud/account-management/e2c-guide/)
```
Comment (chri2547) [inline docs/sources/introduction/grafana-enterprise.md:24]: @ishanjainn , I don't see a migration guide at the location you have specified.

**Review by chri2547 — CHANGES_REQUESTED** [review]
I don't see a guide at the specified link.

**Comment by ishanjainn** [comment]
@chri2547  https://grafana.com/docs/grafana-cloud/account-management/e2c-guide/

We should have one now

File: docs/sources/introduction/grafana-enterprise.md (hunk 1 above)
Comment (ishanjainn) [reply docs/sources/introduction/grafana-enterprise.md:24]: https://grafana.com/docs/grafana-cloud/account-management/e2c-guide/
Just got this merged so should be available

Comment (chri2547) [inline docs/sources/introduction/grafana-enterprise.md:24]:
```suggestion
To migrate to Grafana Cloud, refer to [Migrate from Grafana Enterprise to Grafana Cloud](/docs/grafana-cloud/account-management/e2c-guide/).
```

**Review by chri2547 — APPROVED** [review]
One suggestion. Thank you for the contribution!

**Pull request merged**